docs/source/model_design/concept_model/ensemble_distributions/index.rst

```diff
@@ -49,6 +49,113 @@ fits the data better than any of its individual components.
 Mathematical Definition of Ensemble Distributions
 -------------------------------------------------
 
+Mathematically, an ensemble distribution in GBD is just a `mixture
+distribution`_, which is defined as follows.
+
+Mixture Distributions
++++++++++++++++++++++
+
+Suppose :math:`F_1, F_2,\ldots, F_n` are `cumulative distribution functions
+<CDF_>`_ (CDFs), called the **base distributions** of the ensemble or the
+**components** of the mixture, and suppose we have a collection of non-negative
+numbers :math:`w_1, w_2,\ldots, w_n`, called **weights**, such that
+:math:`\sum_1^n w_i = 1`. Then the **mixture distribution** with components
+:math:`\{F_i\}_1^n` and weights :math:`\{w_i\}_1^n` is the probability
+distribution on :math:`\mathbb{R}` whose cumulative distribution function :math:`F` is defined by
+
+.. math::
+
+  F(x) = \sum_{i=1}^n w_i F_i(x)\quad \text{for } x\in \mathbb{R}.
+
+That is, the CDF of the mixture is just a weighted average of the CDFs of the
+components, with the weight of :math:`F_i` being :math:`w_i`. If the
+:math:`F_i`'s all correspond to `continuous probability distributions`_ on
+:math:`\mathbb{R}`, then differentiating the above equation shows that the
+mixture distribution is also continuous, with `probability density function
+<PDF_>`_ (PDF) given by the coresponding weighted average of the component
+densities, i.e. :math:`f(x) = \sum_1^n w_i f_i(x)`, where :math:`f=F'` and
+:math:`f_i=F_i'`. Thus, the mixture distribution can be defined in terms of
+density functions (PDFs) instead of distribution functions (CDFs) in the
+continuous case.
+
+.. _mixture distribution: https://en.wikipedia.org/wiki/Mixture_distribution
+.. _CDF: https://en.wikipedia.org/wiki/Cumulative_distribution_function
+.. _continuous probability distributions: https://en.wikipedia.org/wiki/Probability_distribution#Continuous_probability_distribution
+.. _PDF: https://en.wikipedia.org/wiki/Probability_density_function
+
+Fitting an Ensemble Distribution to GBD Risk Exposure Data
+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+
+This section describes how GBD uses mixture distributions to define an ensemble
+distribution for modeling risk exposure.
+
+Problem setup
+~~~~~~~~~~~~~~~
+
+Suppose we have a continuous risk exposure variable :math:`E` that we want to
+model. For example, :math:`E` could be body mass index, fasting plasma glucose,
+birthweight, or hemoglobin level. Suppose that we want to model :math:`E` for
+:math:`K` different population groups, e.g. indexed by location/age/sex/year as
+in GBD. Let :math:`k\in \{1,2,\ldots,K\}` specify which population group we are
+referring to. Suppose further that we have microdata on :math:`E` for at least
+some of the groups, and we are able to estimate the mean :math:`\mu_k` and
+standard deviation :math:`\sigma_k` of :math:`E` for each group :math:`k`;
+typically GBD teams do this using ST-GPR or other regression methods. Our goal
```

Comment:
this aside seems out of place here---i suggest you first set up the problem, and then say how GBD researchers typically solve it.

Reply:
I guess there are two problems we could be talking about:

- **First problem:** How do you get means and standard deviations for E for all population groups, given microdata for some population groups, and covariates for all population groups?
- **Second problem:** Once you have means and standard deviations for all population groups, and microdata for some of them, how do you find a distribution to model the exposure of E?

The "problem setup" I'm trying to describe is for the second problem. I just added the comment as an aside about how the first problem is solved so the reader would have some idea how we got to the setup of the second problem. Do you think I should just omit the aside, or should I expand the discussion to include solving the first problem as well as the second problem?

Reply:
I propose you state the first problem, and say that there is a solution (but don't say what it is, or link out to material on ST-GPR); then you can keep your text focused on the second problem.

Reply:
@aflaxman How does it look now?